internal/command/secrets/secrets.go

```diff
@@ -78,8 +78,9 @@ func v1deploySecrets(ctx context.Context, app *api.AppCompact, release *api.Rele
 	return watch.Deployment(ctx, app.Name, release.EvaluationID)
 }
 
-func v2deploySecrets(ctx context.Context, app *api.AppCompact, release *api.Release, stage bool, detach bool) error {
+func DeploySecrets(ctx context.Context, app *api.AppCompact, stage bool, detach bool) error {
```

Comment:
seems not used, is this change needed?

Reply:
Sorry, I didn't push recent changes, and now those changes have changed, so will re-request later!